src/ui/zcl_abapgit_gui_view_repo.clas.abap

```diff
@@ -290,6 +301,10 @@ CLASS zcl_abapgit_gui_view_repo IMPLEMENTATION.
     super->constructor( ).
 
     mo_repo         = zcl_abapgit_repo_srv=>get_instance( )->get( iv_key ).
+    IF mo_repo->is_offline( ) = abap_true.
+      mo_repo->reset_remote( ). " reset on new "show" if ZIP was loaded
+    ENDIF.
```

Comment:
IMHO this is not needed, just let the user decide when to do "Import ZIP" and always let the user have this option

Reply:
Sorry didn't got this point. The idea of this code is: 
- I import zip
- switch to another repo
- and return to the 1st one
- the repo is in "imported" state.

That's kind of strange. I'd expect it to be in initial state. (with Import ZIP button , not Apply). So newly created page forces reset. This is probably not the best pattern ... ideally there should be some `finalize` on `on_page_leave` that should do it. But that mean some refactoring.

Reply:
when switching between online repos, it shows the same state when going back. We have the opportunity to define the behavior, and IMHO it should be as simple as possible, with as few IFs as possible, this will make it easier to maintain